### ISSUE TITLE
Adjustable Delay Between Strings 1 & 2 in String Osc

### ISSUE THREAD

**Issue by ironcross32** [surge-synthesizer/surge#6982]
**Is your feature request related to a problem? Please describe!**

No

A clear and concise description of what the problem is. e.g. I'm always frustrated when [...]

It would be good to be able to adjust the delay between strings 1 and 2 in the string oscillator. This would allow simulation of instruments with a pair of strings tuned to the same or similar pitch, such as a mandolin.

**Describe the solution you'd like:**

I'd like to see the addition of a control that would delay the onset of string 2. This control should also be tempo sync-able.

**Describe alternatives you've considered:**

It's possible to achieve this now with dual mode, string osc in both scenes, taking away string 2 in both cases by putting string balance to 0, then adding a delay with no feedback on scene B.

**Additional context:**

I've been told by several users in the Discord that this isn't possible currently, but that it may be possible in XT2.

**Comment by baconpaul** [comment]
Yeah the main reason we can’t do it is we are out of params. Good idea!

**Comment by baconpaul** [comment]
Good idea for xt2 I mean!
